README.md

```diff
@@ -1,2 +1,12 @@
 # code-katas
 - This repo holds solutions for the Code Wars katas which I have worked on.
+
+## Forbes top 40
+
+This repo features a function which reads an error-ridden json file of 
```

Comment:
error-ridden? That's what we're calling one comma and a couple invalid ages these days?